### PR TITLE
Add message start event

### LINKED ISSUE #737: Add Message Start Event
The message start event should look like this:
Screen Shot 2019-10-10 at 2 39 26 PM

And produce the following BPMN XML:
```xml
<bpmn:startEvent id="node_1">
  <bpmn:messageEventDefinition />
</bpmn:startEvent>
```

Note:
 * It should be green like the other start events.
 * The message start event should have the same _Listen For Messages_ inspector config as the intermediate message catch event.

### PR DESCRIPTION
**Pull request by chipit24**
Fixes #737.

### CODE REVIEW

File: src/assets/toolpanel/Page 1.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="10" viewBox="0 0 16 10" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (lisa-fehr) [inline src/assets/toolpanel/Page 1.svg]: Should the spaces in the filenames be removed? Did you mean to commit the 2 extra files in toolpanel?

File: src/assets/toolpanel/message-start-event.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<svg width="22" height="22" viewBox="0 0 22 22" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (lisa-fehr) [inline src/assets/toolpanel/message-start-event.svg]: end-email-event.svg looks like this:
Screen Shot 2019-10-23 at 11 14 52 AM


Should the envelops in the toolpanel be updated to look all the same?

File: src/assets/toolpanel/Page 1.svg (hunk 1 above)
Comment (chipit24) [reply src/assets/toolpanel/Page 1.svg]: Didn't mean to add this, will remove, thanks!

File: src/assets/toolpanel/message-start-event.svg (hunk 2 above)
Comment (chipit24) [reply src/assets/toolpanel/message-start-event.svg]: Good point; I can update it to keep this consistent.

**Pull request merged**